content/monetization/app-webhooks/index.mdx

```diff
@@ -0,0 +1,61 @@
+---
+weight: 50
+category: "monetization"
+title: "App Webhooks"
+tagline: "Pull data in and out of the Partner Program."
```

Comment:
just a heads up: tagline and description for SEO need to be updated :)

Reply:
Good catch :)